packages/shared/src/components/profile/ProfileSettingsMenu.tsx

```diff
@@ -32,11 +32,14 @@ import { LogoutReason } from '../../lib/user';
 import { useAuthContext } from '../../contexts/AuthContext';
 import { usePrompt } from '../../hooks/usePrompt';
 import { ButtonColor } from '../buttons/Button';
+import { useFeature } from '../GrowthBookProvider';
+import { feature } from '../../lib/featureManagement';
 
 const useMenuItems = (): NavItemProps[] => {
   const { user, logout } = useAuthContext();
   const { openModal } = useLazyModal();
   const { showPrompt } = usePrompt();
+  const showPlusSubscription = useFeature(feature.plusSubscription);
```

Comment:
conditional only when `user.isPlus`, to avoid enrolling users

Reply:
Set it to only when open